com.unity.ml-agents/Runtime/Sensors/ICompressibleSensor.cs

```diff
@@ -0,0 +1,21 @@
+namespace Unity.MLAgents.Sensors
+{
+    /// <summary>
+    /// Sensor interface for generating observations.
+    /// </summary>
+    public interface ICompressibleSensor : ISensor
+    {
+        /// <summary>
+        /// Returns the mapping of the channels in compressed data to the
+        /// actual channel after decompression.
+        /// The mapping is a list of interger index with the same length as
```

Comment:
I think this comment is inaccurate - for the GridSensor it has 8 channels but the mapping length is 12.